qa/workunits/rbd/rbd-nbd.sh

```diff
@@ -244,6 +243,7 @@ rbd-nbd list-mapped | expect_false grep "^${PID} *${POOL} *${IMAGE}"
 # quiesce test
 QUIESCE_HOOK=${TEMPDIR}/quiesce.sh
 DEV=`_sudo rbd-nbd map --quiesce --quiesce-hook ${QUIESCE_HOOK} ${POOL}/${IMAGE}`
+get_pid
```

Comment:
In my merged branch w/ this PR and #37038 it always fails here under teuthology (but not on my machine for some reason):

http://qa-proxy.ceph.com/teuthology/jdillaman-2020-09-10_18:23:09-rbd-wip-jd-testing-distro-basic-smithi/5422637/teuthology.log

Reply:
It looks the problem was with uninitialized `quiesce` variable. I pushed the fix in  #37038.